File: hanghae-cinema/domain/src/main/java/org/example/domain/Reservation.java
package org.example.domain;

import lombok.Builder;
import lombok.Getter;

import java.time.LocalDateTime;

@Getter
@Builder
public class Reservation {
    private Long id;
    private Screening screening;
    private String userId;
    private String seatNumber;
    private LocalDateTime reservedAt;
    private LocalDateTime createdAt;
    private LocalDateTime updatedAt;
    private String createdBy;
    private String updatedBy;
}

File: movie/core/src/main/kotlin/com/example/movie/domain/reservation/model/Reservation.kt
package com.example.movie.domain.reservation.model

import com.example.movie.domain.common.model.BaseModel
import com.example.movie.domain.screening.model.Screening
import com.example.movie.domain.seat.model.Seat
import java.time.LocalDateTime

class Reservation(
    val id: Long = 0,
    val seat: Seat,
    val screening: Screening,
    val userId: Long,
    val reservationTime: LocalDateTime,
    override val createdBy: String,
    override val createdAt: LocalDateTime,
    override val updatedBy: String,
    override val updatedAt: LocalDateTime
) : BaseModel {
}